index.html

```diff
@@ -243,32 +292,36 @@ <h2>Motivation</h2>
       <h2>Use Cases and Requirements</h2>
       <p>
         The Web of Things Interest Group collected use cases for
-        the Web of Things from stakeholders representing various different 
+        the Web of Things from stakeholders representing various different
         industries. This includes both vertical domain-specific use cases and
-        horizontal use cases which apply to multiple application domains 
+        horizontal use cases which apply to multiple application domains
         [[wot-usecases]].
       </p>
       <p>
         Several of the
         <a href="https://www.w3.org/TR/wot-usecases/#sec-vertical-ucs">domain-specific
-        use cases</a> refer to the need for easy integration of 
+        use cases</a> refer to the need for easy integration of
         devices from multiple vendors. This is especially important for
         <a href="https://www.w3.org/TR/wot-usecases/#multi-vendor">cross-domain
-        use cases</a> which require "multi-vendor system 
+        use cases</a> which require "multi-vendor system
         integration" and "out-of-the-box interoperability".
       </p>
       <p>
-        A set of 
+        A set of
         <a href="https://github.com/w3c/wot-profile/blob/main/REQUIREMENTS.md">requirements</a>
-        for Profiles were derived from these use cases. 
+        for Profiles were derived from these use cases.
       </p>
     </section>
 
     <section id="out-of-the-box-interoperability">
       <h2>Out-of-the-Box Interoperability</h2>
+      <p>
+        In layman terms, out-of-box-interoperability implies that every capability of a conformant
+        Thing can be used by a conformant Consumer, within the scope of the profile.
+      </p>
```

Comment:
I'm not sure these are really layman's terms 😆, and the definitions of "profile" and "out-of-the-box interoperability" would depend on each other.

How about...

```suggestion
      <p>
        At a high level, out-of-the-box interoperability means that a Consumer is guaranteed to be able to
        use every capability of a Thing, without Thing-specific customization.
      </p>
```